stock_release_channel_shipment_advice/models/stock_release_channel.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockReleaseChannel(models.Model):
+    _inherit = "stock.release.channel"
+
+    shipment_planning_method = fields.Selection(
+        selection="_get_shipment_planning_method_selection",
+        required=True,
+        default="none",
+    )
+    picking_to_plan_ids = fields.Many2many(
+        comodel_name="stock.picking", compute="_compute_picking_to_plan_ids"
+    )
+    can_plan_shipment = fields.Boolean(compute="_compute_can_plan_shipment")
+    shipment_advice_ids = fields.One2many(
+        comodel_name="shipment.advice",
+        inverse_name="release_channel_id",
+        string="Shipment Advices",
+        readonly=True,
+    )
+
+    @api.model
+    def _get_shipment_planning_method_selection(self):
+        return [("none", "None")] + self.env[
+            "shipment.advice.planner"
+        ]._get_shipment_planning_method_selection()
```

Comment:
you can't rely on this list since some planning methods could requires additional fields.....